src/components/Posts/Posts.tsx

```diff
@@ -0,0 +1,292 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  @media (min-width: 768px) {
+    flex-direction: row;
+  }
+`;
+
+const HeaderTitle = styled.p`
+  color: #266bf3;
+  font-size: 8vw;
+  text-align: center;
+  font-weight: bold;
+
+  @media (min-width: 768px) {
+    font-size: 32px;
+  }
+`;
+
+const PostsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  margin-bottom: 50px;
+
+  @media (min-width: 768px) {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+`;
+
+const FocusedPostContainer = styled.div`
+  width: 100%;
+  height: 431px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 54px;
+  margin-bottom: 45px;
+
+  transition: all 0.3s;
+
+  @media (min-width: 480px) {
+    height: 600px;
+  }
+
+  @media (min-width: 768px) {
+    width: 280px;
+    height: 280px;
+    margin: 20px;
+  }
+
+  @media (min-width: 1200px) {
+    width: 330px;
+    height: 330px;
+  }
+`;
+
+const DescriptionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 54%;
+  width: 100%;
+  padding: 20px 20px;
+  background-color: #eaf1ff;
+`;
+
+const Title = styled.p`
+  color: #266bf3;
+  font-size: 6vw;
+  margin-bottom: 10px;
+  font-weight: bold;
+  text-align: center;
+
+  @media (min-width: 768px) {
+    font-size: 16px;
+  }
+
+  @media (min-width: 1200px) {
+    font-size: 22px;
+  }
+`;
+
+const Description = styled.p`
+  color: #757575;
+  text-align: center;
+  font-size: 3.8vw;
+
+  @media (min-width: 768px) {
+    font-size: 11px;
+  }
+
+  @media (min-width: 1200px) {
+    font-size: 14px;
+  }
+`;
+
+const ReadButton = styled.a.attrs((props: { link: string }) => ({
+  href: props.link,
+}))<{ link: string }>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  height: 30px;
+  margin-top: 12px;
+`;
+
+const BlogButton = styled(ReadButton)`
+  display: none;
+
+  @media (min-width: 768px) {
+    display: flex;
+  }
+`;
+
+const ReadButtonLabel = styled.span`
+  color: #266bf3;
+  font-size: 5vw;
+  margin-bottom: 10px;
+  font-weight: bold;
+  text-align: center;
+
+  @media (min-width: 768px) {
+    font-size: 12px;
+  }
+
+  @media (min-width: 1200px) {
+    font-size: 14px;
+  }
+`;
+
+const StyledFaArrow = styled(FaLongArrowAltRight).attrs({
+  size: 30,
+  color: '#266bf3',
+})`
+  margin-bottom: 12px;
+  margin-left: 8px;
+`;
+
+const ImageContainer = styled.div<{ url: string }>`
+  display: flex;
+  flex-direction: column;
+  height: 46%;
+  width: 100%;
+  background-image: url(${({ url }) => url});
+  background-position: center;
+  background-size: cover;
+`;
+
+const getPostPreview = (text: string) => {
+  const append = text.length >= 107 ? '...' : '';
+  return text.slice(0, 110) + append;
+};
+
+const HeaderIcon = () => (
```

Comment:
Add a return type (gatsby typescript config warns if it's not present)
```suggestion
const HeaderIcon = (): React.ReactElement => (
```